Formula/goproxy.rb

```diff
@@ -25,14 +25,16 @@ def install
   test do
     bind_address = "127.0.0.1:#{free_port}"
     begin
-      server = IO.popen("#{bin}/goproxy -proxy=https://goproxy.io -listen=#{bind_address}")
+      server = IO.popen("#{bin}/goproxy -proxy=https://goproxy.io -listen=#{bind_address}", err: [:child, :out])
       sleep 1
       ENV["GOPROXY"] = "http://#{bind_address}"
-      output = shell_output("go get -v github.com/spf13/cobra 2>&1")
-      assert_match "github.com/spf13/cobra", output
+      test_module = "github.com/spf13/cobra"
+      # Using `system "go", "get", ...` will not get past `brew audit`
+      shell_output("go get #{test_module}")
```

Comment:
Do we just need this to produce an exit code of 0? If so, let's use `system` as in the `install` method instead of `shell_output`.

Reply:
This line triggers the installation of the Go module that goes through the `goproxy` and generates the log output that we then assert against.

As stated in the comment, `system "go", "get", test_module` would be the way to go, but unfortunately `brew audit --strict` is unhappy with that line.

Reply:
Ah, right, of course. What error does `brew audit` produce exactly?

Reply:
```
❯ brew audit --strict goproxy
goproxy:
  * 33: col 7: Do not use `go get`. Please ask upstream to implement Go vendoring
Error: 1 problem in 1 formula detected
```

Reply:
Yes, I forgot about that audit. We should probably fix that by restricting it to the `install` method instead.

Reply:
Sounds like a plan 🙂

Should I already update the test to use `system "go", "get", test_module` or wait until `brew audit` has been updated?

Reply:
Fix c/o @dawidd6 at Homebrew/brew#12428. We can update this when that's merged.